src/utils/system.cpp

```diff
@@ -295,35 +295,36 @@ System::System(int argc, char* argv[])
         if (optarg) axis = atoi(optarg);
         else        axis = 0;
         break;
-      case 'r':   // tube_run
-        if (optarg) tube_run = atoi(optarg);
-        else        tube_run = 1;
+      case 'r':   // official_run
+        if (optarg) official_run = atoi(optarg);
+        else        official_run = 1;
         break;
       case 's':   // elevator_run
         if (optarg) {
           elevator_run = atoi(optarg);
-          tube_run = 0;
+          official_run = 0;
         } else {
           elevator_run = 1;
-          tube_run = 0;
+          official_run = 0;
         }
```

Comment:
if `optarg` is always null then there are a lot of system variables that can be shortened. Are you sure? I thought the point of having it was so that you can write `-fake_imu=false` or smth like that. I agree with having `official_run` outside though, albeit with the caveat that it shows both cases independently. But yes, that should be changed.